src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -85,6 +85,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsEntirelyWithinStringLiteral(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean
```

Comment:
Interpolated string expression does not produce any string literal tokens. Thus this method will return false.
However if the caret would be placed inside of a string found in an interpolation `$" {"|"} "` then the method would return true.